### DISCUSSION TITLE
Option to use /dev/disk/by-id

### DISCUSSION THREAD

**Discussion by harrisonbrose** [dougiteixeira/proxmoxve#296] [Feature request]
Disks sometimes are inconsistent in their /dev/sd_x_ labels. Nodes with multiple disks or hot-swap disks may get more consistent tracking with the option to name the disk entities using their /dev/disk/by-id labels.

**Comment by dougiteixeira** [comment]
I improved this point by using the model in the disk name, together with /dev/sdx in #303.

**Reply by harrisonbrose** [reply]
Thats helpful for cross-referencing manually, but not quite the same - if I move a drive around (or reboot sometimes, even...) my system will re-order the /dev/sdx entries them and so anywhere I'm using the /dev/sdb (for example) entity generated by the integration it will then be referencing the wrong drive and I won't know unless I think to manually check all of the model and seial numbers for each drive and re-reference them everywhere.
